gsa/src/gmp/models/result.js

```diff
@@ -86,7 +87,11 @@ class Result extends Model {
       };
     }
 
-    copy.nvt = Nvt.fromElement(nvt);
+    if (nvt.type === 'nvt') {
+      copy.nvt = Nvt.fromElement(nvt);
+    } else {
+      copy.nvt = Cve.fromResultElement(nvt);
```

Comment:
I'm thinking of changing this var to `copy.information` because `copy.nvt` makes no sense to me anymore.